### DISCUSSION TITLE
notifications not being sent

### DISCUSSION THREAD

**Discussion by JulieVelghe** [solspace/craft-freeform#1298] [Questions & Answers]
### What happened?

Hello,
I have a form that sends 2 notifications, 1 to the person filling in the form, and the other to an admin email address.
The notification to the visitor is being received in their mailbox, however, the notification to the admin is not being received.
I don't see any error being triggered so I'm searching blind.

### How can we reproduce this?

1. make a form
2. set up 2 notifications, one for admin with 1 e-mailadress, and one for the user
3. use file template notifications
4. use Sendgrid as sender

### Freeform Edition

Pro

### Freeform Version

5.2.1

### Craft Version

4.9.4

### When did this issue start?

Unsure

**Comment by kjmartens** [comment]
Hi @JulieVelghe,

Can you check out the "Emails not sending" guide questions here and see if anything applies:
https://docs.solspace.com/craft/freeform/v5/common-issues/#emails-not-sending

If I had to guess, it could be due to one of these common issues:

- A field name is coded incorrectly, e.g. there is something like `{{ email }}` in the template instead of `{{ emailAddress }}`.
- **From Email** address on the email notification template doesn't match the configured email address in the Craft Email Settings area.

If you still cannot resolve this issue, can you share some additional information? 🙂 

1. Screenshot of the form in the builder
2. Screenshot of email notification template or attach its template
3. Check for any errors in the Freeform error log
4. Screenshot of the Freeform Diagnostics page

Thanks! 🙂

**Reply by JulieVelghe** [reply]
Hi, thank you for the feedback :)
I checked the guides, all seems fine, so I'll attach some screenshots.
![image](https://github.com/solspace/craft-freeform/assets/46524074/39013723-6e72-46ab-904e-5b0ffff0e0a0)
![image](https://github.com/solspace/craft-freeform/assets/46524074/412b03e3-3c64-4afe-967d-78a4a92bcf5e)
![image](https://github.com/solspace/craft-freeform/assets/46524074/6379e3b8-e26e-4be3-bc45-c937e3097a97)
![image](https://github.com/solspace/craft-freeform/assets/46524074/a5208d48-4ebe-4922-b100-64699107808c)
![image](https://github.com/solspace/craft-freeform/assets/46524074/f11a6453-7add-44f2-9cd4-f8a63e2a80c6)
[emailtemplates.zip](https://github.com/solspace/craft-freeform/files/15414126/emailtemplates.zip)

**Reply by gustavs-gutmanis** [reply]
Hi @JulieVelghe 

It seems there has been a misunderstanding about the notification templates a bit. It looks like you're trying to use `Globals > Notifications` as your notifications, but Freeform has its own notification templates.

Please go into `Freeform > Notifications > Database Notifications` and edit your notifications there. Take a look at your `Admin` notification and see if there might be anything there that would prevent it from sending.

**Reply by JulieVelghe** [reply]
Hi,
this project has been setup with file templates. Those globals->notifications is a custom field the original dev set up so that the client could easily change the message for notification. The admin notification does not use those globals, and is completely set in the twig file.
I've attached 2 templates (the admin notification and the user notification) in the zip above.

**Reply by gustavs-gutmanis** [reply]
Strange, there seems to be no issues with the templates themselves.
@kjmartens will try to reproduce this and get back to you.

**Reply by JulieVelghe** [reply]
I've removed the notification for the user, in case it was a problem of too much notifications. So now locally, I only have the notification for the admin, but that doesn't seem to help.
I've found something in the web log file, perhaps it is usefull:

``` sh
2024-05-23 15:47:10 [web.INFO] [yii\mail\BaseMailer::send] Sending email "Nieuwe formulier inzending" to "julie@karakters.be" {"memory":53812712} 
2024-05-23 15:47:10 [web.INFO] [application] Request context:
$_GET = [
    'p' => 'nl/contacteer-ons'
]

$_POST = [
    'formHash' => 'ZArDvrDGL-x2w4qlB0Y-leXr3I996dEvz0GcGgRzwVaxPEitTgse16n0WKZL'
    'action' => 'freeform/submit'
    'freeform-action' => 'submit'
    'freeform_form_handle' => ''
    'firstName' => 'Julie'
    'lastName' => 'Velghe'
    'email' => 'julie.velghe@hotmail.com'
    'cellPhone' => '0494700484'
    'address' => 'Provincieweg 19'
    'zipCode' => '9552'
    'city' => 'Borsbeke'
    'subject' => 'Test1'
    'companyName' => 'Test2'
    'Uwvraag' => 'Test3'
    'g-recaptcha-response' => '03AFcWeA65gTSNQ1nWDZ0d1Z3fQKeaY795IvZp0yVGTYCIaYTXlCaYbTkQpVVD5rMVQRDIaX75tlAbn2BlR3lrwTyMy6XpTKd0Ndz0E1J4jhe7BnwtbKEno6ucbx9Hs4ynAo5KdQS2cYzhjdBu9N6PwLOY_shjTlVdxP5PkD4KSVKWdUlvbXHPaqqwbWp8IJuglQ-mH5uHzH6upn0JLrWXKRW5q0PzgKcwvhA7xC993ojCu5JYn3C-JPZGqXNQj72plurfR2xqaSHpLKMfNS0TJS4N-rOXp-Wo5sLXSNM9Xe_pmZnTRmHXPKMIOgo0AP2mg2xAIcZtDl4BHG-dDNUzHdYQ4oRAY-sLYCykzR3yY-541qnTSLBQnfu_T0jpWFzRKfV7bkN-gEqmtEBbD8YdidKZjlW6DhSidwSkvnk9f8ej1mUYubynwclx8qaqnppg6EIymwIgelR1JCKfGrUDSVhlyD85L9WCT57XlWgJ2HSgkHo9zM11-PJKC3-Xc7oJheJEn_NAI-abf5ymLkTep7jio32f1n8KcH8X23O52wJiIOAoEEDqCWG2bW9TqKRyle15fOekVD7sD0t5ezkHz53OwuMWDgMAA-q166ddvbwYJE1K2bq6XVvqF3R8oRIdGlZw04IaUE3qyyr04O92u70YxLTLs_i7QLwprsCAyxf6cXXjOdCWJehX5EJVhmDdfZy-6Y2VEEZC'
    'freeform_payload' => 'V79ogslXo8ZPrPpta5HevjIxOGYxZmZiZTEwZGY4OGIzNGNlMzY1YTA3YWI1YmIwMmEyNTQ4MTY5NjViMWQ4ZjZlZTYwNGYyMjFjZDA3Y2PJkkGuLkj4bDQGsb4yObsaCbVtHKgOKMOJRxTGWJ752JNsx4SVuVPJJOSzXE9ZJ2vbQf8KAFMSrL0+Zmnp6uCMCWRL+P0obOH9oebj4JXf3GShE0QM548APfih87NITHM6kEEK+ILrDYvyVl8ZUPX462i4zojMSbE/EFzJJIxm1eKISaCPSk7I5/tpKCWhEm7zkEShwWatZ3ufRB27nLicxj1O0tKx2ls4PMCMqIKt512p6b4G06+TuATYtTHLNl1guNEgY4fIt4hRMYRs0+jVT+FaTzChunR9BZ6hmJzOL45U8Dqe7m0xDAiExV//QUFqxUBofP/Di1NFv/r3o1hFXZ9Tfv4+T+kinY0rqhKFS65wAm7ZFtWJCbDXptdXPxzhcKV/IobIdNX2u3jFQ660bXvKT8AIR4Wyzat61z1J/Nw/whSCYGoR6VlRcFX4Lspig5nlRWvP/cOW1xzP/WnaJE5XQOQdfeOYb+iGQqS1juk8B7nwAfUFFeTtHv7FAc6t1oOsA8eFrWH2gwYZTvImnVJ+Rl5WoXf5rS3BUrhO7d217DujtZyBe2e5g5Se5eCnvN9ok3cMzpz9GeSeUkVXr2nUVSrXbiULlPSOukBoAPKE+u0kexxHnk68VN/A4Usfr/+nHYECPvT7OPRlgCEL9p9qeeD/m/wsz2Cvt9F1rVpPFx4rcFB+W/qt/tJVXsItnmgSk4OQW2/8sD6IHyCKWSYRpSrCDH8jCNIQX1YYZPG/GZGs6cB6oHkcx0Pwn1wyT5Z/7/4QD0b3+hsVm3pHwBbaFg9bSH3BzJIrg9VPwh/5TLJCdRulDhzdZvOk0mlfHWiCJmiNDZg/acd6Iq0ven1ksGu2zOV6ZRJqsBYG4nUpuk/MdnLfP6jw7WyskCcIr9iIw/ZXF/3px3ub6YdkjH7FaURGS/jgKUrCHYHbNLAfRn1gM4oeHek96mMw90riEq5pDroDbbFgKO2m1nE09Nobdx5gUc0ZyLkCv80Jh8MFSRWOl8bV8FnworrYFEphyAAbx/dc9sZk6BHgDBSygXMBjq1rGMTzP5h9QNgovlavMoVWKFzclJFHX8Q+NdQ='
]

$_FILES = []

$_COOKIE = [
    'bfcaaa754345db03b99012bb4211ef54_username' => '3f67e5cee31b77eed1d643694a4c911bd1791cba663c3ddc47923db3f278b04aa:2:{i:0;s:41:\"bfcaaa754345db03b99012bb4211ef54_username\";i:1;s:6:\"benoit\";}'
    'CRAFT_CSRF_TOKEN' => '••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••'
    'CraftSessionId' => '8fee47d77ca70d96706b5a5260cbea58'
    'bfcaaa754345db03b99012bb4211ef54_identity' => 'dea9a12bc8b6ea0eeaee0c8aecff4fa99bc5e42a01e92af2923f01464950fad5a:2:{i:0;s:41:\"bfcaaa754345db03b99012bb4211ef54_identity\";i:1;s:159:\"[1,\"[\\\"isTAxRL-zZ0OKx8AHGg2OHFf6WB9iT4EkuoXElDcJBbwmOyLwpsD5dU8LKmStEPBEKfZRI-HbhCPEDt8u-kPcPXsmN8tL391AvwX\\\",null,\\\"a16ddaab909d2cf27fce353f26dd2ff2\\\"]\",3600]\";}'
    'form_posted_1' => 1716472030
]

$_SESSION = [
    '6490340e19179ffc3af8029cfc9e7fa1__flash' => [
        'freeform_submission_flash' => -1
    ]
    'freeform_submission_flash' => 1
    'bfcaaa754345db03b99012bb4211ef54__token' => '••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••'
    'bfcaaa754345db03b99012bb4211ef54__id' => 1
    '__authKey' => '••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••'
    'bfcaaa754345db03b99012bb4211ef54__expire' => 1716475628
    '__duration' => 3600
]
```

**Comment by JulieVelghe** [comment]
This issue is no longer a problem. Inspecting Sendgrid revealed that the notifications were being send, further digging revealed that the got stuck in the quarantine.
